keyboards/rico/phoenix_project_no1/rules.mk

```diff
@@ -0,0 +1 @@
+WS2812_DRIVER = vendor
```

Comment:
```suggestion
```
Should be moved to info.json as `ws2812.driver: vendor`

Reply:
Added ws2812.driver: vendor in info.json.
Will apply your suggestion.